src/styles/utils/_variables.scss

```
@@ -0,0 +1,10 @@
+$nav-height: 60px;
+$card-width: 200px;
+$page-content-margin-bottom: 50px;
+$margin-between-cards: 48px / 2;
+$catalog-width: $card-width * 4 + $margin-between-cards * 8;
```

You have the same variable in another file. Let's not duplicate it. Remove it from one of them
```suggestion
$margin-between-cards: 48px / 2;
```